exampleSite/config.toml

```diff
@@ -35,7 +35,7 @@ social_banner = "img/banner.png"
 usesmallsummarycard = true
 posts_navigation = true
 # issoHost = "comments.domain.tld:1234"
-# githubRepo = "githubUsername/repositoryName"
+# githubRepo = "githubUsername/repositoryName/tree/master"
```

Comment:
Because this is required, everyone who currently uses the feature but doesn't notice this change will have their version break. Hmm...